scripts/quests/sandoria/A_Job_for_the_Consortium.lua

```diff
@@ -0,0 +1,177 @@
+-----------------------------------
+-- A Job for the Consortium
+-- Glenne - Southern Sandoria, !pos -122 -2 15 230
+-- Aaveleon - West Ronfaure, !pos -431 -45 343 100
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/items')
+----------------------------------
+local ID = require("scripts/zones/Port_San_dOria/IDs")
+----------------------------------
+local quest = Quest:new(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.A_JOB_FOR_THE_CONSORTIUM)
+
+quest.reward =
+{
+    gil = 1000,
+    fame = 25,
+    fameArea = xi.quest.fame_area.NORG,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getQuestStatus(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.THE_BRUGAIRE_CONSORTIUM) == QUEST_COMPLETED and
+                player:hasKeyItem(xi.ki.TENSHODO_MEMBERS_CARD) and
+                player:getFameLevel(xi.quest.fame_area.SANDORIA) >= 5
+        end,
+
+        [xi.zone.PORT_SAN_DORIA] =
+        {
+            ['Portaure'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'AIRSHIP_BLOCK') < os.time() then
```

Comment:
prog, Stage, Option vars

Reply:
I don't understand the need to change the var name here. stage, option, or other names would be too vague imo. This would increase readability, no?

Reply:
readability to a GM perspective and a gm command that can pull all vars for a specific quest
if everyone knows the vars used for all quests since they are uniform then it makes debugging faster